drivers/ovsdriver.go

```diff
@@ -341,12 +423,68 @@ func (d *OvsDriver) CreateEndpoint(id string) error {
 	return nil
 }
 
-// DeleteEndpoint deletes an endpoint by named identifier.
-func (d *OvsDriver) DeleteEndpoint(id string) (err error) {
+//UpdateEndpointGroup updates the epg
+func (d *OvsDriver) UpdateEndpointGroup(id string) error {
+	var (
+		err error
+	//	intfName string
+	)
+	//gets the EndpointGroupState pbject
+	cfgEp := &mastercfg.EndpointGroupState{}
+	cfgEp.StateDriver = d.oper.StateDriver
+	err = cfgEp.Read(id)
+	if err != nil {
+		return err
+	}
+
+	var EpgBandwidth uint64
 
+	if cfgEp.ID != "" {
+		cfgEpGroup := &mastercfg.EndpointGroupState{}
+		cfgEpGroup.StateDriver = d.oper.StateDriver
+		if err != nil {
+			log.Errorf("error converting bandwidth %v, err: %v", cfgEpGroup.Bandwidth, err)
+			return err
+		}
+		err = cfgEpGroup.Read(cfgEp.ID)
```

Comment:
Why are we repeating the read here? Didn't we read the epg state on line 435 already?